GLTF/include/GLTFNode.h

```diff
@@ -71,6 +71,9 @@ namespace GLTF {
 
 		Transform* transform = NULL;
 
+        // Special implementation that takes a predicate so mappings can be updated for all children
```

Comment:
The spacing here looks off.

I know we do have some spacing inconsistencies throughout the project. I'm working on adding some automatic clang-tidy stuff to CI to resolve this, but for now, could you match what is in the file here?